docs/packages.md

```diff
@@ -132,9 +132,41 @@ Example:
 
 ![New file package route](assets/pics/qs-file-packages/file-package-new-route.png)
 
-## What’s Next?
+## Event Collection Packages
 
-- [Create a Pixel](/web-traffic)
-- [Learn more about the Scarf Gateway](/gateway)
+Event Collection Packages on Scarf are another flexible package type which can be used to collect event telemetry by fetching a URL or by importing event history. Event Collection Packages are an alias of File packages and share all the same traits. Common use cases include:
+
+- Sending custom telemetry or other events from your application
+- Importing log or event history from your hosted installations to Scarf
+
+### Creating an Event Collection Package
+1. Once signed in to Scarf, navigate to the home page.
+
+2. Click plus icon in the navigation, then select New Package.
+![Create a new package](assets/pics/qs-file-packages/create-new-package.png)
+
+3. In the first drop-down click on the package type you would like to create. For this section you will click `Event Collection`.
+![Create a package](assets/pics/qs-file-packages/create-file.png)
+
+4. Select the package owner from the dropdown.
+![Select package owner](assets/pics/qs-file-packages/file-package-select-owner.png)
+
+5. Give your package a name.
+![Name your package](assets/pics/qs-file-packages/file-package-name.png)
+
+### Adding an Incoming URL
+This section explains what the Outgoing and Incoming URLs are and how to use a URL template.
```

Comment:
We don't have outgoing URLs in this context actually. We also may want to call out that you don't _need_ to use public gateway URLs, eg if you are only going to use the bulk import API.